lib/index.js

```diff
@@ -45,12 +46,21 @@ Elevio.prototype.identify = function(identify) {
   var name = identify.name();
   var email = identify.email();
   var plan = identify.proxy('traits.plan');
+  var traits = identify.traits();
+
+  var removeTraits = ['id', 'name', 'firstName', 'lastName', 'email'];
+
+  removeTraits.forEach(function(traitItem) {
+    if (traits && traitItem in traits) {
+      delete traits[traitItem];
+    }
+  });
 
   var user = {};
   user.via = 'segment';
   if (email) user.email = email;
   if (name) user.name = name;
   if (plan) user.plan = [plan];
-
+  if (Object.keys(traits).length > 0) user.traits = traits;
```

Comment:
@duellsy We can't rely on the all browsers having `Object.keys` (IE9+ only).

Here's a component you can use instead:

https://github.com/ndhoule/keys